.github/workflows/ci.yaml

```diff
@@ -29,6 +29,26 @@ jobs:
         run: yarn test
       - name: Integration Test
         run: yarn test:integration
+  test-build:
+    runs-on: ubuntu-18.04
+    if: github.ref != 'refs/heads/master'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Detect Dockerfile change
+        id: changed-dockerfile
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+            Dockerfile
+      - name: Build test
+        if: steps.changed-dockerfile.outputs.any_changed == 'true'
+        uses: docker/build-push-action@v4
+        with:
+          load: true
+          tags: ghcr.io/elifesciences/enhanced-preprints:test
+      - name: Test docker run
+        if: steps.changed-dockerfile.outputs.any_changed == 'true'
+        run: docker run --rm ghcr.io/elifesciences/enhanced-preprints:test
```

Comment:
CI seems to be hanging on this. I will try and recreate locally.

Reply:
I'm guessing it's because I have to run with the detached flag and then I could perhaps hit localhost:3000 and check I get 200?